File: Python/1000-1099/1012.py
A, B, C = [float(x) for x in input().split()]

a_ = A*C/2

b_ = C**2*3.14159

if A > B:
    c_ = (B*C) + ((A-B)/2*C)

else:
    c_ = (A*C) + ((B-A)/2*C)

d_ = B**2

e_ = A*B

print(f'TRIANGULO: {a_:.3f}\n'
    f'CIRCULO: {b_:.3f}\n'
    f'TRAPEZIO: {c_:.3f}\n'
    f'QUADRADO: {d_:.3f}\n'
    f'RETANGULO: {e_:.3f}')

File: C++/1000-1099/1012.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main(){
    double A, B, C, TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO, pi = 3.14159;

    cin >> A >> B >> C;

    TRIANGULO = (A*C)/2;
    CIRCULO = pi * pow(C, 2.00);
    TRAPEZIO = (A+B)/2 * C; 
    QUADRADO = B*B;
    RETANGULO = A*B;

    cout << fixed;
    cout.precision(3);
    cout << "TRIANGULO: " << TRIANGULO << endl;
    cout << "CIRCULO: " << CIRCULO << endl;
    cout << "TRAPEZIO: " << TRAPEZIO << endl;
    cout << "QUADRADO: " << QUADRADO << endl;
    cout << "RETANGULO: " << RETANGULO << endl;
    return 0;
}
